src/dbnode/storage/cleanup.go

```diff
@@ -178,13 +172,7 @@ func (m *cleanupManager) WarmFlushCleanup(t time.Time, isBootstrapped bool) erro
 	return multiErr.FinalError()
 }
 
-func (m *cleanupManager) ColdFlushCleanup(t time.Time, isBootstrapped bool) error {
-	// Don't perform any cleanup if we are not boostrapped yet.
-	if !isBootstrapped {
-		m.logger.Debug("database is still bootstrapping, terminating cleanup")
-		return nil
-	}
-
+func (m *cleanupManager) ColdFlushCleanup(t time.Time) error {
```

Comment:
Don't you think this is changing the semantics of this logics? Previously, the method was returning early in cases when `m.database.IsBootstrapped()` (passed via `isBootstrapped` argument) was `true`. Now, you only check `s.IsBootstrapped()`, but from what I see, `m.database.IsBootstrapped()` is no longer respected.